tokens/global/color.json5

```diff
@@ -4,9 +4,8 @@
     "docexample": "color",
     "text": {
       "primary": {
-        "$value": "{options.color.warm-grey-1150}",
+        "$value": "{options.color.warm-grey-1150-75}",
```

Comment:
I'm ok with this for now but feel that we would get more bang for our buck by moving alpha to its own token and delivering these as a composite 
it would allow for the adjustment of alpha of all color and reduce the amount of color options we needed to support